source/heavy/computing.rst

```diff
@@ -65,7 +65,8 @@ Using Compute Canada for the first time
 
         # User specific aliases and functions
 
-        module load java/1.8.0_192 singularity/3.5
+        module load java/11.0.2
```

Comment:
I looked at one of my recent sbatch and it starts with
```
module load nextflow/23.10.0 apptainer/1.3.4
```
And I don't have any other `module load` in my .bashrc so I think that's all you need.